cerebrium/cortex/advanced-functionality/streaming.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Streaming Endpoints"
+---
+
+Streaming allows users to stream live output from their models using server-sent event (SSE) streams. This works for Python objects which implement the
+iterator protocol which is anything that essentially uses the 'yield' command in Python. You can return any content types as long as it is returned as a string
+
+<Note>
+  This feature is currently in beta and so if you would like to stream output,
+  please replace '**predict**' in your endpoint url with '**stream**'
+</Note>
+
+Let us see how we can implement a simple example below:
+
+```python
+def predict(item, run_id, logger):
+    for i in range(10):
+        yield f"Number {i} "
+        time.sleep(1)
+```
+
+Once you deploy this code snippet and hit the stream endpoint, you will see the SSE events progressively appear. The latest Postman has great functionality to show this.
```

Comment:
Nice docs so far. 
One small suggestion is to give an example of what hitting the “stream endpoint” looks like. 
Minor but there’s no harm in giving a user an example of what the new url would look like so that they know that you’re meaning they must make a post request to that endpoint